tools/IoTEdgeDevOps/DevOpsLib/IoTEdgeRelease.cs

```diff
@@ -42,17 +46,20 @@ public class IoTEdgeRelease
 
         public string Name => this.name;
 
+        public string SourceBranch => this.sourceBranch;
+
         public VstsReleaseStatus Status => this.status;
 
         public Uri WebUri => this.webUri;
 
         public int NumberOfEnvironments => this.environments.Count;
 
-        public static IoTEdgeRelease Create(VstsRelease vstsRelease) =>
+        public static IoTEdgeRelease Create(VstsRelease vstsRelease, string branchName) =>
```

Comment:
why the argument name is "branchName", not "sourceBranch"?

Reply:
I kept it consistent with this:
https://github.com/philipktlin/iotedge/pull/2/files#diff-2824a82f05ba86dd5d44c578f5d26033L32.

It is better to be consistent within the class so I will make this change.